features/problem.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Error handling valid according to RFC 7807 (application/problem+json)
+  In order to be able to handle error client side
+  As a client software developer
+  I need to retrieve an RFC 7807 compliant serialization of errors
+
+  Scenario: Get an error
+    When I add "Accept" header equal to "application/json"
+    And I send a "POST" request to "/dummies" with body:
+    """
+    {}
+    """
+    Then the response status code should be 400
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/problem+json"
+    And the JSON should be equal to:
+    """
+    {
+       "type": "https://tools.ietf.org/html/rfc2616#section-10",
```

Comment:
So this should be:

```json
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1"
```

p/s: The indentation in this section is wrong (3 spaces instead of the intended 4)